.github/workflows/github_pages.yml

```diff
@@ -0,0 +1,32 @@
+name: Github Pages -> Algolia Crawler
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  algolia_recrawl:
+    name: Algolia Recrawl
+    runs-on: ubuntu-latest
+    steps:
+      # checkout this repo
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+      # checkout the private repo containing the action to run
+      - name: Checkout GitHub Action Repo
+        uses: actions/checkout@v2
+        with:
+          repository: algolia/algoliasearch-crawler-github-actions 
+          ref: v0.4.2
```

Comment:
Is renovate able to upgrade that number automatically ?

Reply:
I guess it's somehow possible but TBH I don't really know how to do it. Maybe it would be easier to try to get the latest version instead ?

Reply:
on main branch yes, but after that we execute using `./` so I'm not even sure what it's doing